### PR TITLE
Implement getFirebaseProject and deprecate firebaseApi

### PR DESCRIPTION
**Pull request by TrCaM**
### Description

- Implement `getFirebaseProject` in `projects.ts` that fetch information of a Firebase project given `projectId` as input
- Deprecate `firebaseApi`, and replace code that use `firebaseApi` with API methods from `management/projects.ts`
- Update unit tests

### Testing
- Manual testing with `firebase init` and `firebase use` commands, which were users of `firebaseApi`

### Next step
- Update choosing projects from a list behavior:
     + List a single page of maximum of 100 projects
     + If next page token is provided then prompt user for `projectId` string
- Update [create a new project] option while choosing a project of `firebase init` to actually create a new Firebase project and choose it as the working project for the directory. 
- Deprecate `firebase list` command

### CODE REVIEW

**Comment by TrCaM** [comment]
I hope that I covered all uses of `firebaseApi` in the commands. Please inform if you discover any broken command.

File: src/init/features/project.ts [hunk 1]
```diff
@@ -40,9 +43,9 @@ export async function getProjectInfo(options: any): Promise<ProjectInfo> {
  * @returns A {@link FirebaseProject} object.
```
Comment (nbegley) [inline src/init/features/project.ts]: This javadoc `@returns` line looks out-of-date and unhelpful, so let's just remove it.

File: src/init/features/project.ts [hunk 2]
```diff
@@ -63,8 +66,8 @@ async function selectProjectFromOptions(options: any): Promise<ProjectInfo> {
...
```
Comment (nbegley) [inline src/init/features/project.ts]: Same for this `@returns` line.

**Comment by TrCaM** [comment]
Closing this PR, replace with a new PR that has the latest updates.

**Pull request closed without merging**